### PR TITLE
LPS-112871 - Replace usages of aui:col with clay:col

### PR DESCRIPTION
**Pull request by marcoscv-work**

### CODE REVIEW

**Comment by marcoscv-work** [comment]
ci:test:sf

**Comment by marcoscv-work** [comment]
ci:test:stable

**Comment by jbalsas** [comment]
This should have test failures since the `id` attribute is not properly namespaced.

See the following commits in the original PR:
- [LPS-112626 Namespace ids since new tag better follows our attribute convention.](https://github.com/brianchandotcom/liferay-portal/pull/88326/commits/551a686171678595bbb6628e3736b0072bc427a8)
- [LPS-112870 Namespace ids since new tag better follows our attribute convention.](https://github.com/brianchandotcom/liferay-portal/pull/88326/commits/2179cd9aba2cd1b5d726a6d3c767a0065cd880d9)

> In general, when tags accept an `id` attribute, we set its value as is without
any modifications. We often default `id` to `portletNamespace`+`name` when the tag also
supports a `name` attribute and an explicit `id` is not set.
> 
> For some reason, `aui:container` didn't follow this pattern and namespaced id when
used in the context of a portlet.
> 
> For coherence, the new `clay:container` tag does not namespace the `id` attribute,
so we need to namespace such usages when updating from `aui:container`

I'll take care of that and retest.

**Comment by marcoscv-work** [comment]
ci:test:sf

**Comment by marcoscv-work** [comment]
ci:test:stable

File: modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/edit_calendar_booking.jsp [hunk 1]
```diff
@@ -388,41 +388,51 @@ while (manageableCalendarsIterator.hasNext()) {
 						<clay:row
 							className="calendar-booking-invitations"
 						>
-							<aui:col width="<%= (calendarBooking != null) ? 25 : 33 %>">
+							<clay:col
+								md="<%= (calendarBooking != null) ? String.valueOf(3) : String.valueOf(4) %>"
```
Comment (jbalsas) [inline modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/edit_calendar_booking.jsp:392]: I think we'll need to SF these occurrences to:

```
String.valueOf((calendarBooking != null) ? 3 : 4)
```

**Comment by marcoscv-work** [comment]
ci:stop

**Comment by marcoscv-work** [comment]
ci:test:stable

**Comment by jbalsas** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
ci:forward

**Comment by jbalsas** [comment]
I think this is good to go, we can push straight to Brian since the `relevant` failures are clearly unrelated.

I'd like to wait for @eduardoallegrini, though, so we can send just one PR with everything together to reduce risk of regressions. If I can get @eduardoallegrini's part before 16PM today, we should be able to push it all together. @eduardoallegrini, what do you think?

**Comment by eduardoallegrini** [comment]
I have the replacements here https://github.com/eduardoallegrini/liferay-portal/pull/69/, but I need to solve the conflicts with today's master, add the imports of the new taglibs into the modules and run SF

**Comment by wincent** [comment]
> I have the replacements here [eduardoallegrini#69](https://github.com/eduardoallegrini/liferay-portal/pull/69), but I need to solve the conflicts with today's master, add the imports of the new taglibs into the modules and run SF

How you going with the conflicts? Need any help?

**Comment by eduardoallegrini** [comment]
https://github.com/eduardoallegrini/liferay-portal/pull/70
- Rebased to master ✅
- Solved conflicts ✅
- SF ✅ 
- Adding "imports" to modules ⏳60%

**Comment by wincent** [comment]
I started the tests on your fork @eduardoallegrini, so we'll see if they're green by the time Chema gets back. In any case, he'll have to fetch that commit and combine it anyway so I don't know whether he'll want to do another run after that.

**Comment by jbalsas** [comment]
Hey @eduardoallegrini, yeah, just add the missing imports and send it my way.

SF just tells you what you're missing in https://github.com/eduardoallegrini/liferay-portal/pull/70#issuecomment-624629692

```
     [java] 1: Missing taglib for tag with prefix 'clay', see https://github.com/liferay/liferay-portal/blob/master/modules/util/source-formatter/src/main/resources/documentation/checks/jsp_missing_taglibs_check.markdown: ./modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/layout/view.jsp (SourceCheck:JSPMissingTaglibsCheck)
     [java] 2: Missing taglib for tag with prefix 'clay', see https://github.com/liferay/liferay-portal/blob/master/modules/util/source-formatter/src/main/resources/documentation/checks/jsp_missing_taglibs_check.markdown: ./modules/apps/comment/comment-web/src/main/resources/META-INF/resources/edit_discussion.jsp (SourceCheck:JSPMissingTaglibsCheck)
     [java] 3: Missing taglib for tag with prefix 'clay', see https://github.com/liferay/liferay-portal/blob/master/modules/util/source-formatter/src/main/resources/documentation/checks/jsp_missing_taglibs_check.markdown: ./modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_renderer/page.jsp (SourceCheck:JSPMissingTaglibsCheck)
     [java] 4: Missing taglib for tag with prefix 'clay', see https://github.com/liferay/liferay-portal/blob/master/modules/util/source-formatter/src/main/resources/documentation/checks/jsp_missing_taglibs_check.markdown: ./modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/asset/full_content.jsp (SourceCheck:JSPMissingTaglibsCheck)
     [java] 5: Missing taglib for tag with prefix 'clay', see https://github.com/liferay/liferay-portal/blob/master/modules/util/source-formatter/src/main/resources/documentation/checks/jsp_missing_taglibs_check.markdown: ./modules/apps/gogo-shell/gogo-shell-web/src/main/resources/META-INF/resources/view.jsp (SourceCheck:JSPMissingTaglibsCheck)
     [java] 6: Missing taglib for tag with prefix 'clay', see https://github.com/liferay/liferay-portal/blob/master/modules/util/source-formatter/src/main/resources/documentation/checks/jsp_missing_taglibs_check.markdown: ./modules/apps/image-uploader/image-uploader-web/src/main/resources/META-INF/resources/view.jsp (SourceCheck:JSPMissingTaglibsCheck)
```

Just add those and then it should be good to go.

> In any case, he'll have to fetch that commit and combine it anyway so I don't know whether he'll want to do another run after that.

Yeah, I'm hoping that `green` + `green` almost-equals `green`...  so here's hoping🤞

**Comment by eduardoallegrini** [comment]
I did those changes but the `ant all` is still failing

**Comment by jbalsas** [comment]
> I did those changes but the ant all is still failing

You added both the `tld` declaration in the `jsp` and the module dependency in the `build.gradle`?

What's the message of the error?

Send it to me if you're still struggling with it to see if I can push it the last mile

**Comment by eduardoallegrini** [comment]
>You added both the tld declaration in the jsp and the module dependency in the build.gradle?

Yes

>Send it to me if you're still struggling with it to see if I can push it the last mile

Let me fix the errors you found in https://github.com/eduardoallegrini/liferay-portal/pull/70

**Comment by jbalsas** [comment]
Ok, so still halfway through the review at https://github.com/eduardoallegrini/liferay-portal/pull/70

Let's take it easy on that one and get this one moving.

@wincent, @john-co, can you guys please make sure this makes it in?

Thanks!! 😘

**Comment by wincent** [comment]
Manual forward here: https://github.com/brianchandotcom/liferay-portal/pull/88377

Includes promise to:

> fast-follow with any fixes for any fallout, but we've asked @john-co to confirm our analysis.

**Comment by wincent** [comment]
ci:close

**Comment by john-co** [comment]
The failing tests look unrelated. I'll perform some other checks to confirm those issues are tracked.

**Comment by john-co** [comment]
The DisplayPageTemplateServiceTest failures are already tracked by: 
- https://issues.liferay.com/browse/LRQA-58410
- https://issues.liferay.com/browse/LRQA-58411

For Histograms test, created https://issues.liferay.com/browse/LRQA-58460
cc/ @rodrigocunhaa

**Pull request closed without merging**